src/Symfony/Component/Workflow/MarkingStore/MarkingStoreInterface.php

```diff
@@ -36,8 +36,7 @@ public function getMarking($subject);
     /**
      * Sets a Marking to a subject.
      *
-     * @param object  $subject A subject
-     * @param Marking $marking A marking
+     * @param object $subject A subject
```

Comment:
need a doc update, no?

Reply:
Why ?

Reply:
i mean why removing $marking doc param where you should have added the new one $context?
i do not understand

Reply:
There is a type hint on the marking. PHP docs was telling the same. It was useless. So i removed it.